ArtOfIllusion/src/artofillusion/RotateViewTool.java

```diff
@@ -154,18 +160,18 @@ private void dragRotateTravelSpace(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown && !e.isShiftDown())
+		if (mouseButtonTwo(e) && controlDown)
```

Comment:
```suggestion
		if (controlDown && !e.isShiftDown())
```
It should be possible to use this block by MB1 too, when the tool is selected by the icon. Forcing MB2 here prevents z-rotate in the case of icon-selected tool.

Also you can not be in this part of the code if you pressed MB3, so the detection is obsolete.

The `!e.isShiftDown()` is necessary to separate this case from the next, when the shift-key actually is down.

Reply:
Got it... Perhaps the cases should be reversed?

Not going to happen today, though...